animeal/src/Flows/Main/Modules/More/MoreContract.swift

```diff
@@ -0,0 +1,65 @@
+import UIKit
+
+// MARK: - View
+protocol MoreViewable: AnyObject {
+    func applyActions(_ viewItems: [MoreItemView])
+}
+
+// MARK: - ViewModel
+typealias MoreViewModelProtocol = MoreViewModelLifeCycle
```

Comment:
also you can use parentheses to make the following format:
typealias MoreViewModelProtocol = (
   MoreViewModelLifeCycle &
   ... &
   ...
)